R-package/R/lgb.Dataset.R

```diff
@@ -353,7 +357,8 @@ Dataset <- R6::R6Class(
       if (!lgb.is.null.handle(private$handle)) {
 
         # Get feature names and write them
-        cnames <- lgb.call.return.str("LGBM_DatasetGetFeatureNames_R", private$handle)
+        cnames <- lgb.call.return.str(fun_name = "LGBM_DatasetGetFeatureNames_R"
+                                      , private$handle)
```

Comment:
```suggestion
        cnames <- lgb.call.return.str(
            fun_name = "LGBM_DatasetGetFeatureNames_R"
            , private$handle
        )
```

can you please make this indentation match the rest of the package and the description in #3390 ?

Reply:
Yeah, I am interested. I'll claim all the rest of files!